README.adoc

```diff
@@ -228,7 +248,7 @@ inventory service. Register this client with a `JsrJsonpProvider` JSON-P provide
 to process JSON resources. The `teardown()` method destroys this client instance.
 
 The following list describes the test cases that you can run:
-* Use the `testGetPropertiesTime()` test case to test the `@Timed` annotation. This test case sends a request to the `http://localhost:9080/inventory/systems/localhost` URL. The test verifies the `200` response code, which is returned with the `connectToEndpoint()` private method. Next, the test case makes a connection to the `http://localhost:9080/metrics/application` URL to get the content of the
+* Use the `testGetPropertiesTime()` test case to test the `@Timed` annotation. This test case sends a request to the `http://localhost:9080/inventory/systems/localhost` URL. The test verifies the `200` response code, which is returned with the `connectToEndpoint()` private method. Next, the test case makes a connection to the `https://localhost:9443/metrics/application` URL to get the content of the
```

Comment:
There should be a blank line before this for the list to properly show up.

Reply:
Done